libraries/AC_Fence/AC_Fence.cpp

```diff
@@ -108,6 +109,13 @@ const AP_Param::GroupInfo AC_Fence::var_info[] = {
     // @User: Standard
     AP_GROUPINFO_FRAME("AUTOENABLE", 10, AC_Fence, _auto_enabled, static_cast<uint8_t>(AutoEnable::ALWAYS_DISABLED), AP_PARAM_FRAME_PLANE),
 
+    // @Param: ALT_FRAME
```

Comment:
We should probably name this parameter ALT_TYPE to be consistent with existing parameters named RTL_ALT_TYPE and FOLL_ALT_TYPE regardless of whether "frame" or "type" is a better word, it would at least keep us consistent.